### DISCUSSION TITLE
discussion

### DISCUSSION THREAD

**Discussion by augustoaraujoo** [Your-First-Open-Source-Project/Your-First-Open-Source-Project#40] [Ideas]
Hello, how will open source projects work, can I submit some projects to the "Your-First-Open-Source-Project" community to help me? , I have a project to help backend programmers by facilitating a part of the development , I published the project on npm

-- project in npm : https://www.npmjs.com/package/repo-express-example-node

**Comment by melvincwng** [comment]
@augustoaraujoo, Hi augusto, you can refer to this page over [here](https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project) for more information on how this beginner-friendly open-source project works.

With regards to your query on the submission of open source projects, we are not accepting user projects actually.

Hope this clarifies your question.

Once again, let me warmly welcome you to our organization 'Your First Open Source Project' and hope to see your first PR in our organization soon :)

**Reply by augustoaraujoo** [reply]
thank you so much
